modules/shared/src/main/scala/org/tessellation/security/signature/Signed.scala

```diff
@@ -140,7 +140,7 @@ object Signed {
         InvalidSignatureForHash(signed).asLeft[Hashed[A]].pure[F]
       )
 
-    def toHashedWithSignatureCheck[F[_]: Async: Hasher: SecurityProvider](
+    def toHashedWithSignatureCheckEnsuringSerializer[F[_]: Async: Hasher: SecurityProvider](
```

Comment:
Why? It already has a different definition. What does it mean "ensuring serializer"? Serializer is being provided at execution

Reply:
I forgot to remove, updating